staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/deepcopy.go

```diff
@@ -244,5 +244,15 @@ func (in *JSONSchemaProps) DeepCopy() *JSONSchemaProps {
 		}
 	}
 
+	if in.XMapType != nil {
+		in, out := &in.XMapType, &out.XMapType
+		if *in == nil {
```

Comment:
I just realized that this condition is pointless (`*in` can't be `nil` because of `if` two lines above). This pattern is also repeated dozens of times in this file :-/

Reply:
thanks for pointing that, changed.
I'll be happy to put a follow up PR to clean that up for other fields, I'm also wanting to move up to constants all the scattered string raw references to marks and types